### ISSUE TITLE
UI Redesign: Work with "dark mode" browser settings.

### ISSUE THREAD

**Issue by PaulBone** [mozilla/hubs#3557]
**Description**
When using a firefox plugin (and possibly other ways to acheive the same effect, eg browser settings) to adjust the colours used by pages to force all pages into a dark mode.  Some controls become hidden.

![Screenshot_2020-12-15 Youthful Studious Tract Mozilla Hubs](https://user-images.githubusercontent.com/3138117/102146196-ac466380-3ebc-11eb-8ea3-df32c9cf6200.png)

In the screen shot my background colour is dark, I can see a circle above each of the Invite and Voice labels, but I can't see what's in the circle (which is not a problem, I can still click it).

The panel on the right is also missing some icons (I think there's a close icon on the top left of this panel, my mouse pointer changed when I hovered there, I clicked it and the panel went away).

When I joined the room and was invited to test my sound. I saw a circle for "Talk to test Microphone". I didn't see anything inside it, and I thought I had to click it but that seemed to do nothing. It took me a few moments to notice it was reacting to sounds in my office with a green fill effect. D'oh!  It literally says "Talk to test Microphone" and when I spoke it reacted, but I might not have noticed if my microphone was muted, since it never would have reacted.  Maybe this is a seperate bug? the problem was that I didn't know the circle was a sound meter.

**To Reproduce**
1. Using firefox
2. Install https://addons.mozilla.org/en-US/android/addon/dark-background-light-text/
3. Open the UI redeisgn version of hubs.

**Hardware**
 - Device: x86_64 Laptop, external screen
 - OS: Linux Mint
 - Browser: Firefox Nightly 85.0a1 (2020-12-06) 

**Additional context**
Here's an article I wrote about my eyesight and how web accessability affects me:
https://paul.bone.id.au/blog/2017/11/26/how-i-see-the-web/

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-526)
